### PR TITLE
Publish AC Dev 2.1.1

### PR DESCRIPTION
**Pull request by ernest-kr**
Publish AC Dev 2.1.1

I have created a new AC dev version 2.1.1 for testing

### CODE REVIEW

File: 2.1.1/buster/build-time-pip-constraints.txt [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+apache-airflow-providers-google==3.0.0
+apache-airflow-providers-imap==1.0.1
+apache-airflow-providers-microsoft-azure==2.0.0
+apache-airflow-providers-mysql==1.1.0
+apache-airflow-providers-postgres==1.0.2
+apache-airflow-providers-redis==1.0.1
+apache-airflow-providers-slack==3.0.0
+apache-airflow-providers-sqlite==1.0.2
+apache-airflow-providers-ssh==1.3.0
+apispec==3.3.2
```
Comment (kaxil) [inline 2.1.1/buster/build-time-pip-constraints.txt:20]: Does this come from https://github.com/apache/airflow/blob/constraints-2-1/constraints-3.7.txt ?

Comment (ernest-kr) [reply 2.1.1/buster/build-time-pip-constraints.txt:20]: can i replace the entire contents of it here with 2-1contraints?

Comment (ernest-kr) [reply 2.1.1/buster/build-time-pip-constraints.txt:20]: @kaxil

Comment (kaxil) [reply 2.1.1/buster/build-time-pip-constraints.txt:20]: Yeah

**Review by danielhoherd — APPROVED** [review]
Seems good to me.

File: 2.1.1/buster/build-time-pip-constraints.txt (hunk 1 above)
Comment (ernest-kr) [reply 2.1.1/buster/build-time-pip-constraints.txt]: Done

File: 2.1.1/buster/build-time-pip-constraints.txt [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+apache-airflow-providers-apache-hive==1.0.3
+apache-airflow-providers-apache-kylin==1.0.1
+apache-airflow-providers-apache-livy==1.1.0
+apache-airflow-providers-apache-pig==1.0.1
+apache-airflow-providers-apache-pinot==1.0.1
+apache-airflow-providers-apache-spark==1.0.3
+apache-airflow-providers-apache-sqoop==1.0.1
+apache-airflow-providers-celery==1.0.1
+apache-airflow-providers-cloudant==1.0.1
+apache-airflow-providers-cncf-kubernetes==1.2.0
```
Comment (kaxil) [inline 2.1.1/buster/build-time-pip-constraints.txt:61]:
```suggestion
apache-airflow-providers-cncf-kubernetes==1!1.2.1
```

fixes our Istio issue -- for all AC release we will have to use this

**Review by kaxil — APPROVED** [review]

**Pull request merged**